blog/templates/blog/blogbase.html

```diff
@@ -38,6 +38,7 @@
         <!--  <a class="blog-nav-item" href="#">New features</a> -->
           <a class="blog-nav-item" href="{% url "about_view" %}">About</a>
           <a class="blog-nav-item" href="{% url "blog_view" %}">Blog Home</a>
+          <a><i class="fa fa-bed"></i></a>
```

Comment:
``<a>`` elements are used for links.  This doesn't appear to be linking to anything.  If you don't want a link try a different element like a ``<span>``.